### DISCUSSION TITLE
Support Commerce Order Importing

### DISCUSSION THREAD

**Discussion by engram-design** [engram-design/FeedMe#890] [Ideas]
Must include addresses, customers, users, transactions

**Comment by RitterKnightCreative** [comment]
On the [element types](https://docs.craftcms.com/feed-me/v4/content-mapping/element-types.html) page of the documentation, it's mentioned that orders for Commerce are supported but I assume this functionality is coming soon since there doesn't seem to be a way to import them at this point?

**Comment by migmake** [comment]
It is marked enhancement but I did not see orders element listed in my feed...

**Comment by angrybrad** [comment]
@RitterKnightCreative @migswd looks like order support hasn't been completely implemented yet. Have updated the element type docs to reflect it.

**Comment by BenParizek** [comment]
This would be a very helpful feature in selling commerce projects. Right now, due to limited documentation on how this is possible and the number of complex relationships/details that need to be imported accurately under the hood, the costs of doing this manually are often too high to add to existing Commerce project costs.

While I'd be the first to not trust anecdotal evidence, I've personally been involved in at least 3 Commerce projects that would have been easier to sell and would have resulted in better initial websites, if we could have included historic order data via a supported import process.

**Comment by aryehischechter** [comment]
Has any progress been done on this? It would be very helpful to have this functionality.

**Comment by billmn** [comment]
Yes please, orders import would be really helpful!
